src/mono/wasi/runtime/CMakeLists.txt

```diff
@@ -14,8 +14,11 @@ target_compile_options(dotnet PUBLIC @${NATIVE_BIN_DIR}/src/wasi-default.rsp @${
 set_target_properties(dotnet PROPERTIES COMPILE_FLAGS ${CONFIGURATION_WASICC_FLAGS})
 
 target_link_libraries(dotnet
-    #TODOWASI ${ICU_LIB_DIR}/libicuuc.a
-    #TODOWASI ${ICU_LIB_DIR}/libicui18n.a
+    ${CMAKE_SYSROOT}/lib/wasm32-wasi/libc++.a
```

Comment:
I wonder why this is not automatic.

Reply:
I'm not sure myself. I wondered the same thing.

Reply:
The project is defined as `project(mono-wasi-runtime C)`. I hit the same thing, and adding stdc++/libc++ reference is the easy fix. It may be possible to change the `project` to use `CXX` as the language but I didn't try that.